rewrite-maven/src/test/java/org/openrewrite/maven/java/UseJavaxNullabilityAnnotationsTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.maven.java;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.config.Environment;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.*;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+/**
+ * <Beschreibung>
+ * <br>
+ *
+ * @author askrock
+ */
```

Comment:
Can we also remove this comment?

Please also note that your contribution will in addition to the release notes also be attributed in the Moderne SaaS (as extracted from the Git history), as you can see in the bottom of this screenshot :-) 
<img width="518" alt="image" src="https://user-images.githubusercontent.com/408765/230323193-e9fbd770-272d-4e18-8109-4852bec56670.png">

Reply:
I did not mean to commit that, that's how it was configured on my other device, sorry about that. 🙈